docs/user/rocket/rocket_axes.rst

```diff
@@ -0,0 +1,145 @@
+   argument of the class constructor :class:`rocketpy.Rocket.__init__`. This
+   argument defines the direction of the axis that follows the rocket's center
+   line. It can be either ``"nose_to_tail"`` or ``"tail_to_nose"``.
+
+.. seealso:: 
+     
+    See `Positions and Coordinate Systems <positions.rst>`_ for more
+    information on how this is used in the definitions of the rocket's
+    aerodynamic surfaces and motor.
+
+The ``x`` and ``y`` axes are defined in the plane perpendicular to the center axis,
+and the ``z`` axis is defined along the center axis. Depending on the choice of
+``coordinate_system_orientation``, the ``x`` axis and ``y`` axis can be inverted.
+
+The following figure shows the two possibilities for the user input coordinate system:
+
+.. figure:: ../../static/rocket/3dcsys.png
+  :align: center
+  :alt: Rocket axes
+
+.. note::
+
+    When ``coordinate_system_orientation`` is set to ``"tail_to_nose"``, the direction
+    of the ``x``, ``y``, and ``z`` axes of the **User Defined Coordinate System** is
+    the same as the **Body Axes Coordinate System**. The orgin of the coordinate
+    system may still be different.
+
+Angular Position Inputs
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Angular position inputs (``angular_position``) refer to the roll angle position
+of that surface along the rocket's tube. The roll angle is defined as the angle 
+from the ``y`` axis to the surface. Currently, only :class:`rocketpy.RailButtons`
+use this input.  
+
+The following figure shows the roll angle
+definition for both ``coordinate_system_orientation`` options:
+
+.. figure:: ../../static/rocket/angularpos.png
+  :align: center
+  :alt: Angular position
+
+
+.. note::
+
+   The positive direction of the roll angle is defined as the direction that 
+   rotates the surface in the positive direction of the ``z`` axis.
+
+
+
+2. Body Axes Coordinate System
+------------------------------
+
+The body axes coordinate system is used inside the simulation to assess the
+governing equations of motion. The body axes coordinate system is defined as follows:
+
+- The origin is at the rocket's center of dry mass (``center_of_dry_mass_position``).
+- The ``z`` axis is defined along the rocket's centerline, pointing from the 
+  center of dry mass towards the nose.
+- The ``x`` and ``y`` axes are perpendicular.
+
+3. Relation to Flight Coordinates
+---------------------------------
+
+The ``Flight`` class uses a coordinate system defined as follows:
+
+- The origin is at the launch rail.
+- The ``Z`` axis is positive upwards.
+- The ``X`` axis is position eastwards.
+- The ``Y`` axis is positive northwards.
+
+The following figure shows the rotational relationship between the
+**Body Axes Coordinate System** and the **Flight Coordinate System**:
+
+.. figure:: ../../static/rocket/flightcsys.png
+  :align: center
+  :alt: Flight coordinate system
+
+In the figure $\bf{i}$** is the ``inclination`` and $\bf{h}$ is the
```

Comment:
Use .rst syntax instead of latex

```suggestion
In the figure **:math:`i`** is the ``inclination`` and $\bf{h}$ is the
```

Also applies to other places;